assets/.gitignore

```
@@ -2,6 +2,7 @@
 *.c
 !text/*.c
 *.h
+!text/*.h
```

```suggestion
*.h
*.c
!text/*.h
!text/*.c
```
this is really splitting hairs here